.github/workflows/ci.yml

```
@@ -160,7 +160,7 @@ jobs:
       cmake-args: -DBUILD_MQT_CORE_BENCHMARKS=ON -DBUILD_MQT_CORE_MLIR=ON -DBUILD_MQT_CORE_BINDINGS=ON
       clang-version: 21
       build-project: true
-      files-changed-only: true
+      files-changed-only: false
```

Before merging, this should be switched back to `true`.

```suggestion
      files-changed-only: true
```